routes/api/v1/playlists.js

```diff
@@ -51,23 +51,27 @@ router.delete('/:id', async function (request, response) {
   }
 });
 //
-// router.post('/', async function (request, response) {
-//   try {
-//
-//     if (!('title' in request.body)) {
-//       return response.status(404).json({"error": "You must include a title parameter in the request"});
-//     }
-//     let data = await favoritesPresenter.newFavorite(request.body)
-//     if (data == undefined) {
-//       return response.status(400).json({"error": "Record not created. Song with that title could not be found"});
-//     } else {
-//       return response.status(201).json(data);
-//     }
-//   }
-//   catch(error) {
-//     return response.status(500).json({"error": "Request could not be handled"});
-//   }
-// });
+router.post('/', async function (request, response) {
+  try {
+
+    if (!('title' in request.body)) {
+      return response.status(400).json({"error": "You must include a title parameter in the request"});
+    }
+    if (request.body.title === '') {
```

Comment:
Nice counting for this sad path! 

We might need something similar for the favorites POST endpoint. For a refactor, what about creating a helper method that validates the title? The method could valid that both body title exist and the title value is not blank.